ddtrace/sampler.py

```diff
@@ -52,7 +62,8 @@ class RateSampler(BaseSampler):
     It samples randomly, its main purpose is to reduce the instrumentation footprint.
     """
 
-    def __init__(self, sample_rate=1):
+    def __init__(self, sample_rate=1.0):
```

Comment:
Can we do this change separately? There are further usages below that use `int`s

Reply:
Sounds fair to me, I'll address this in a separate PR then 👍